tasks/localizationtasks.js

```diff
@@ -149,15 +144,64 @@ gulp.task('ext:localization:xliff-to-ts', function () {
 		.pipe(gulp.dest(config.paths.project.root + '/src/constants/'));
 });
 
+// Generates a package.nls.json file from localizedPackage.json.enu.xlf
```

Comment:
It would be nice to give a brief description of how it accomplishes that. i.e. what are the input files, and what does it do to them to get the output files? (that can be done as more thorough comments in the function itself too)